qdrant-landing/content/articles/hybrid-search-revamped.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "Hybrid Search Revamped"
+short_description: "Merging different search methods to improve the search quality was never easier"
+description: "Qdrant 1.10 introduces a new Query API to build a search system that combines different search methods to improve the search quality."
+preview_dir: /articles_data/hybrid-search-revamped/preview
+social_preview_image: /articles_data/hybrid-search-revamped/social-preview.png
+weight: -150
+author: Kacper Łukawski
+author_link: https://kacperlukawski.com
+date: 2024-07-21T14:24:00.000Z
+---
+
+It's been more than a year since we published the [original article](/articles/hybrid-search) about building a hybrid 
+search system with Qdrant. The idea was straightforward: combine the results from different search methods to improve 
+the search quality. Back in 2023, you were still expected to use an additional service to bring the lexical search 
+capabilities and combine all the intermediate results. Things have changed since then. Once we introduced the support of 
+the sparse vectors, [the additional search service became obsolete](/articles/sparse-vectors/), but you were still 
+required to combine the results from different methods on your side.
+
+**Qdrant 1.10 introduces a new Query API that allows you to build a search system that combines different search methods 
+to improve search quality**. Everything is now done on the server side, and you can focus on building the best search 
+experience for your users. In this article, we will show you how to utilize the new [Query 
+API](/documentation/concepts/search/#query-api) to build a hybrid search system.
+
+## Introduction to the new Query API
+
+At Qdrant, we believe that vector search capabilities go well beyond the simple search for the nearest neighbors.
+That's why we provided separate methods for different search use cases, such as `search`, `recommend`, or `discover`.
+With the latest release, we are happy to introduce the new Query API, which combines all of these methods into a single 
+endpoint and also supports creating nested multistage queries that can be used to build complex search pipelines.
+
+If you are an existing Qdrant user, you probably have a running search mechanism that you want to improve, either sparse 
+or dense. Your system will act as a baseline for further experiments, so you have a reference point to compare the new
```

Comment:
This is kind of unclear.

Reply:
Changed to "If you are an existing Qdrant user, you probably have a running search mechanism that you want to improve, whether sparse or dense. Doing any changes should be preceded by a proper evaluation of its effectiveness. " and moved the evaluation paragraph just below this sentence. The idea was to make the need for evaluation more prominent.